### PR TITLE
runtimetest: add linux default symbolic link validation

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by Mashimiao** [comment]
ping @opencontainers/runtime-tools-maintainers

**Comment by liangchenye** [comment]
LGTM

File: cmd/runtimetest/main.go [hunk 1]
```diff
@@ -341,6 +348,29 @@ func validateLinuxDevices(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateDefaultSymlinks(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux default Symbolic links")
```
Comment (mrunalp) [inline cmd/runtimetest/main.go]: nit: Symbolic -> symbolic

Comment (Mashimiao) [inline cmd/runtimetest/main.go]: fixed, thanks

**Comment by Mashimiao** [comment]
@mrunalp @liangchenye  PTAL

**Comment by hqhq** [comment]
LGTM

**Pull request merged**